### PR TITLE
Handbrake 1.2.2: add recipe and patchset

### PR DESCRIPTION
**Pull request by kenmays**
Built on hrev53102. Disabled for further dev review.
NOTES: 
- Need to cleanup older libbluray version on x86_gcc2.
- Use Feature.nvenc=0 in build/GNUMakefile (no GPU hw accel on Haiku).
- Utilize BeOS/Haiku GUI work from kallisti5 (?)

### CODE REVIEW

**Comment by extrowerk** [comment]
PR is broken on 64 bit

File: media-video/handbrake/patches/handbrake-1.2.2.patchset [hunk 1]
```diff
@@ -0,0 +1,3820 @@
...
++ScanWindow::ScanWindow( hb_handle_t * handle )
++    : BWindow( BRect( 0,0,10,10 ), "Scan", B_FLOATING_WINDOW_LOOK,
++               B_MODAL_SUBSET_WINDOW_FEEL, B_NOT_CLOSABLE |
++               B_NOT_RESIZABLE | B_NOT_ZOOMABLE )
++{
++    /* Add the scan view */
++    fView = new ScanView( handle );
++    AddChild( fView );
++    
++    /* Resize to fit
```
Comment (extrowerk) [inline media-video/handbrake/patches/handbrake-1.2.2.patchset]: Why is this required?

**Comment by extrowerk** [comment]
After adding 3 missing dependency to get it fail at missing "libxml/parser.h" i think we should close this.

@kenmays: You can open a new if you got it built locally. It would be great, if you could add a screenshot to your future PRs which shows that your recipe built without problem locally, and generated the packages.

So something like this: http://0x0.st/zcYT.png (so not 100% or anything else, but that the packages actually generated).

**Comment by kenmays** [comment]
Adding CPPFLAGS fixed libxml2 issue (I set it for my build test). Mainly wanted to know about the Handbrake GUI concerns to add to this or just drop it.
